core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -311,6 +312,29 @@ public void format() throws IOException {
         .toString());
   }
 
+  /**
+   * Triggers a checkpoint in this ufs journal.
+   */
+  public synchronized void checkpoint() throws IOException {
+    long nextSequenceNumber = getNextSequenceNumberToWrite();
+    if (nextSequenceNumber == getNextSequenceNumberToCheckpoint()) {
+      LOG.info("{}: No enough journal entries to create checkpoint.",
```

Comment:
We can be more specific, saying that no entries have been written since the last checkpoint

Reply:
Changed, more clear. thanks